README.adoc

```diff
@@ -68,3 +68,38 @@ The Keep Go client.
 
 It runs the Keep network’s random beacon and participates in related
 computations.
+
+== Quick installation
+
+To quickly install and start a single client use the installation
```

Comment:
It is not necessarily about a single client. One can start as many clients as they want with those scripts.

Reply:
Hm, not sure if a `single client` means _only 1 client_ in this context. I would interpret it as a single client at a time, cause the script won't run multiple clients after execution. 

Anyway, I removed "single" just to avoid any confusion.